### DISCUSSION TITLE
Swagger Compatiblity

### DISCUSSION THREAD

**Discussion by glatzert** [Badgerati/Pode#730] [General]
Hi Matthew,

I've put the swagger file into https://editor.swagger.io/ and found some errors, which were reported.
I started looking into the existing warnings and started making code changes to have them fixed.

If you like to have it, I'll provide them as PR as well (wanted to do a heads up this time).
Essentially it's:
 - not having a `requestBody` in GET and DELETE (that's forbidden)
 - not having empty `format` (it's default)
 - removing `enum = null` (also forbidden)

**Comment by Badgerati** [comment]
Very nice!  Please, feel free to open a PR 😄 

It wouldn't surprise me if there were a few more like that, the errors I fixed during implementing were from swagger errors on the frontend.

**Comment by swedan333** [comment]
'New-PodeOAStringProperty -pattern' does not add 'pattern' to openapi json file!

**Comment by swedan333** [comment]
Add-PodeRoute -Path '/users/:userId' 
adding a path variable produces an error (using editor.swagger.ui to verify openapi output)
Declared path parameter "userId" needs to be defined as a path parameter at either the path or operation level
